src/Integration/Constants.cs

```diff
@@ -27,11 +27,16 @@ public static class Constants
         /// <summary>
         /// The property key which corresponds to the Roslyn analyzer additional files
         /// </summary>
-        public const string AdditionalFilePropertyKey = "AdditionalFile";
+        public const string AdditionalFilesPropertyKey = "AdditionalFiles";
```

Comment:
Strictly speaking this isn't a property: "AdditionalFilesItemTypeName" / "AdditionalFilesItemTypeKey" would be better.